chalice/app.py

```diff
@@ -160,26 +162,55 @@ class MultiDict(Mapping):
     def __init__(self, mapping):
         if mapping is None:
             mapping = {}
+        elif isinstance(mapping, list):
+            dict_mapping = {}
+            for item in mapping:
+                k, v = item
+                if k in dict_mapping:
+                    dict_mapping[k].insert(0, v)
+                else:
+                    dict_mapping[k] = [v]
+            mapping = dict_mapping
 
         self._dict = mapping
+        self._remapped_keys = set()
 
-    def __getitem__(self, k):
-        values_list = self._dict[k]
+    def add(self, k, v):
```

Comment:
what's this method for?

Reply:
Ah a stub I forgot. Initially it was going to be https://werkzeug.palletsprojects.com/en/0.15.x/datastructures/#werkzeug.datastructures.MultiDict.add

Since there is no other way to add a new value to an internal list, other than getting it, appending and resetting. Initially I was going to use it during `__init__` when creating the internal structure from a list of tuples. However it was awkward since the internal map hasn't been initialized so I opted to write it all out there in the `__init__` method so there is no real reason to have it, since it didn't exist before. I just forgot to remove the stub.